### PR TITLE
[fix bug 1235539] 'Other releases' menu link on release notes page is not keyboard accessible

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

File: media/js/firefox/releasenotes.js [hunk 1]
```diff
@@ -11,16 +11,44 @@
...
+        $subMenuTitle.on('focus', function() {
+            $subMenu.attr('aria-expanded', true);
+        });
+
+        $subMenuTitle.on('click', function(e) {
+            e.preventDefault();
+        });
+
+        $subMenuLinks.on('focusout', function(e) {
+            if ($(e.target).parent().is('li:last-child')) {
```
Comment (jpetto) [inline media/js/firefox/releasenotes.js]: Small issue here - if you tab backwards/shift-tab from the last `<li>`, the menu disappears. Maybe needs a `$subMenuLinks.on('focus')` handler?

Comment (alexgibson) [inline media/js/firefox/releasenotes.js]: Hmm, good catch! A `focus` event handler doesn't seem to work here, so I think the solution will require a `keydown` event in order to pick up the shift key.

**Comment by alexgibson** [comment]
Thanks @jpetto - updated.

**Comment by jpetto** [comment]
Sweet, sweet keyboard navigation. :candy:

r+!

**Pull request merged**